documentation/java.asciidoc

```diff
@@ -11,19 +11,37 @@ The arguments (`devon java «args»`) are explained by the following table:
 |=======================
 |*Argument(s)*                       |*Meaning*
 |`setup`                             |setup OpenJDK (install or update and verify), link:configuration.asciidoc[configurable] via `JAVA_VERSION` (e.g. `8u242b08` or `11.0.6_10`)
-|create «args»                     |create a new Java project based on link:https://github.com/devonfw/devon4j/blob/master/documentation/tutorial-newapp.asciidoc[devon4j application template]. If a single argument is provided, this is the package name and is automatically split into `groupId` and `artifactId`. Use `-DdbType=«db»` to choose the database (hana, oracle, mssql, postgresql, mariadb, mysql, h2, hsqldb). Any option starting with dash is passed as is."
+|`create «args»`                     |create a new Java project. You can choose between Quarkus and Spring. Spring creation is based on https://github.com/devonfw/devon4j/blob/master/documentation/tutorial-newapp.asciidoc[devon4j application template]. If a single argument is provided, this is the package name and is automatically split into groupId and artifactId for Spring. To select a framework use the -f or -framework parameter with the chosen framework. Use `-DdbType=«db»` to choose the database for both frameworks (hana, oracle, mssql, postgresql, mariadb, mysql, h2, hsqldb). Any option starting with dash is passed as is." 
 |`migrate [from «version»] [single]` |migrate a https://github.com/devonfw/devon4j[devon4j] project to the latest version. If for some reasons the current devonfw version can not be auto-detected you may provide it manually after the 'from' argument. Also the 'single' option allows to migrate only to the next available version."
 |`cicd «args»`                       |generate cicd files for the current devon4java project
 |=======================
 
 == create
+With the create command you can choose to create a Spring or a Quarkus application. 
```

Comment:
With the create command, you can choose to create a Spring or Quarkus application
OR
You can choose to create a Spring or Quarkus application with the create command.